src/fides/api/graph/config.py

```diff
@@ -493,6 +497,13 @@ def identities(self) -> Dict[FieldPath, str]:
             if field.identity
         }
 
+    def custom_request_fields(self) -> Dict[FieldPath, str]:
```

Comment:
utility to know whether a collection has custom_request_fields. I followed the same format as the `identities` function to try to keep it consistent

Reply:
If I didn't say this elsewhere, great docstring here now 👍